project_template/src/operations.c

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#include "operations.h"

```

Comment:
>"operations.h" [](start = 9, length = 14)

Is this intended to be used like this?

It seems that the project folder structure is crafted to provide a namespace-like construct.

Is this intended to actually be #include "az/exampleshortname/operations.h"?